### ISSUE TITLE
还是出现pred_wh not finite问题。

### ISSUE THREAD

**Issue by WYQ-Github** [duwt/OrienMask#15]
很感谢您的工作，我在测试和评估都没遇到问题。
在默认设置，训练的时候出现pred_wh not finite。我查看issue，Increasing the learning rate warm-up steps from 500 to 1000 makes the training procedure more stable. 但是还是会报错。之后我又把warm-up的step改为500，将config中设置(n_gpu=1, batch_size=8, accumulate=2还是会出现一样的错误。

pytorch1.8+cuda11.1，py=3.8 单卡

**Comment by duwt** [comment]
目前的结论是单卡跑很容易出现not finite，可选的方案是降低orien损失的权重，改为15甚至10，但性能上会有一定下降。我们初期实验都是单卡在跑，后面pytorch和cuda版本有升级，代码也有重构，暂不确定是什么因素导致了单卡无法正常训练。

**Comment by WYQ-Github** [comment]
好的。谢谢回复，期待后续的代码更新。
